README.md

```diff
@@ -9,13 +9,13 @@
 
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://join.slack.com/t/fugue-project/shared_invite/zt-jl0pcahu-KdlSOgi~fP50TZWmNxdWYQ)
 
-**Fugue is an abstraction layer that helps big data practitioners accelerate development, decrease costs, and simplify maintenance of their big data projects.** This is done by allowing users to port Python, pandas, and SQL code to Spark and Dask with minimal changes, making it easy to leverage distributed computing.
+**Fugue is a unified framework for distributed computing that lets users port Python, pandas, and SQL code to Spark and Dask with minimal changes.** 
```

Comment:
this is complained by codacy: `[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading`

So it thinks this all bold section should be headline. I think that doesn't make sense. But it's their rule. We may need a workaround.

Reply:
Ah I got this error before. The workaround is to either do the **bold** sentence followed by unbolded so that it's not treated as a header.

Or you really just need to use header 3 maybe? I think we can immediately follow it with a non-bolded sentence

Reply:
It also complained about header 3 unfortunately. No choice but to add a sentence after that bolded section.